SPECS/espeak-ng/espeak-ng.spec

```diff
@@ -0,0 +1,77 @@
+Summary:        Compact text-to-speech synthesizer
+Name:           espeak-ng
+Version:        1.50
+Release:        1%{?dist}
+# Apache2 license applies only to Android APK code- does not apply here
+# BSD license applies only to Windows code- does not apply here
+License:        GPLv3 and Unicode
```

Comment:
Nit: I think "and" and "or" are supposed to be all-caps in the `License` tag.

Reply:
Resolved, thanks